CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [1.6.0] - Unreleased
 
-- Added: [#5484](https://github.com/ethereum/aleth/pull/5485) aleth-bootnode now by default connects to official Ethereum bootnodes. This can be disabled with `--no-bootstrap` flag.
+- Added: [#5485](https://github.com/ethereum/aleth/pull/5485) aleth-bootnode now by default connects to official Ethereum bootnodes. This can be disabled with `--no-bootstrap` flag.
```

Comment:
I believe this also should be `Changed:` as it is not a new feature.

Reply:
Why not, this is new code and new flag added

Reply:
Can be. It rather feels to me we are changing only how features are enabled / disabled. But I don't care much.